Cabal/Distribution/Types/GenericPackageDescription.hs

```diff
@@ -61,6 +65,9 @@ data Flag = MkFlag
 
 instance Binary Flag
 
+emptyFlag :: FlagName -> Flag
```

Comment:
I'll not repeat the code in the haddock. Comments are IMHO for explaining *why* definition exists or *how" it's meant to be used, not *what* it is (there is "source" for that).
In this case it should be irrelevant, and I unfortunately don't know why default parameters are what they are (but would rather comment that in code, not in haddock).

So it's just ` -- | A 'Flag' initialized with default parameters .`